lib/requestIterator.js

```diff
@@ -28,7 +29,7 @@ RequestIterator.prototype.nextRequestBuffer = function () {
 
 RequestIterator.prototype.move = function () {
   // get the current buffer and proceed to next request
-  const ret = this.currentRequest.requestBuffer
+  const ret = new Buffer(this.currentRequest.requestBuffer.toString().replace(/\[<uuid>\]/g, uuid()))
```

Comment:
this is where we lose speed. Can we do this change using a function passed in as an option? In this way we could support more scenarios that just `:id:`.